backend/src/main/java/com/jobwait/spring/SpringEndpoints.java

```diff
@@ -0,0 +1,60 @@
+package com.jobwait.spring;
+
+import java.util.function.Supplier;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.jobwait.control.RequestController;
+import com.jobwait.fault.FaultException;
+import com.jobwait.spring.utils.Utils;
+
+@CrossOrigin("http://127.0.0.1:3000")
+@RestController
+public class SpringEndpoints {
+	private static ResponseEntity<ObjectNode> processAndHandleException(Supplier<ObjectNode> func) {
+		ObjectNode responseNode = Utils.mapper.createObjectNode();
+		responseNode.set("data", null);
+		responseNode.set("error", null);
```

Comment:
what is the use of these two lines? i would prefer it if you when you got an error there is no associated data since its null, it gives me no info. and vice-versa, if i got data i didnt get an error so i dont have to check it.

Reply:
just saw your long comment on this, but i still believe its an odd thing to get an error and also potentially get data, explicit for me would be i see error key so its an error and i see no error theres no error.

Reply:
Yeah I can remove this, I was on the fence about it so if you prefer it a certain way, I don't mind.

I interpret it as a question/answer pair, like "data?" and if you see null, there's no data, same with "error?", null means all good.

Reply:
If you get an error, you will never get data, same for data, so maybe that's an argument that these keys represent a dichotomy and so shouldn't both be in the response body